Boolean.CSharp.Main/Account.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Boolean.CSharp.Main
+{
+    public class Account
+    {
+        private string name;
+        private List<Transaction> transactions;
+        private float balance;
+        public Account(string name) 
+        {
+            this.name = name;
+            transactions = new List<Transaction>();
+        }
+
+        public bool deposit(float amount, string date)
+        {
+            balance += amount;
+            Transaction trans = new Transaction(balance, date, amount, "credit");
```

Comment:
no need to pass date; the Transaction() constructor could simply compute today's date when it is created